libs/brain-nrf52840/board.json

```diff
@@ -21,20 +101,49 @@
     "buttons": []
   },
   "gpioPinMap": {
+    "MISO": "MISO",
+    "MOSI": "MOSI",
+    "SCK": "SCK",
+    "NEOPIXEL": "NEOPIXEL",
```

Comment:
you need to specify which neopixel pin you want to be used

Reply:
The one named neopixel is ok. Doesn't really matter for this board.